doc/contributing/CODING_STANDARDS.md

```diff
@@ -404,6 +404,30 @@ So for types that are converted to/from native Python types, it is safe
 to assign them as default argument values.  Note that this includes
 e.g. `std::vector` (Python `list`), which is _not_ safe in pure Python.
 
+### The GIL
+
+The GIL should be released for most non-trivial bound methods using
```

Comment:
[Consider] "Most non-trivial" isn't helpful, should define why only "most" or just leave it off.
```suggestion
The GIL should be released for non-trivial bound methods using
```

Reply:
E.g. https://github.com/OpenAssetIO/OpenAssetIO/pull/1144#discussion_r1360740141 .  Also things like `TraitsData`, `EntityReference`,`SeverityFilter`, `HostSession`, all have methods that don't release the GIL, because they can't be overridden and are trivial.

Reply:
Apologies, I skipped over the `[Consider]` and just addressed the `[Question]`.